app/presenters/speedy_af/proxy/admin/collection.rb

```diff
@@ -44,6 +44,8 @@ def to_param
   end
 
   def cdl_enabled?
+    # Cast to boolean since it is indexed as a string (`_ssim`)
+    cdl_enabled = cdl_enabled&.to_b
```

Comment:
Instead of casting to boolean, would it be better to index it as a boolean in the first place? I didn't think about it when we originally set it up, but looking at our solr schema there are a few boolean options. I think `cdl_enabled_bsi` should work?

Also, did we have a reason for storing it as `_ssim` rather than `_ssi`? I'm not sure that cdl_enabled needs to be getting handled like a multi-value field.

Reply:
I wasn't sure about this.  Changing how it is indexed would require us to reindex existing collections so I took the approach of making it work with the current data.  But it would be better to index as `_bsi` and that shouldn't be too onerous to ask as part of the upgrade.  I'll change this.